src/integrationTest/java/org/opensearch/security/SystemIndexTests.java

```diff
@@ -39,13 +46,17 @@ public class SystemIndexTests {
     public static final AuthcDomain AUTHC_DOMAIN = new AuthcDomain("basic", 0).httpAuthenticatorWithChallenge("basic").backend("internal");
 
     @ClassRule
-    public static final LocalCluster cluster = new LocalCluster.Builder().clusterManager(ClusterManager.SINGLENODE)
+    public static final LocalCluster cluster = new LocalCluster.Builder().clusterManager(ClusterManager.DEFAULT)
         .anonymousAuth(false)
         .authc(AUTHC_DOMAIN)
         .users(USER_ADMIN)
-        .plugin(ExampleSystemIndexPlugin.class)
+        .plugin(List.of(SystemIndexPlugin1.class, SystemIndexPlugin2.class))
         .nodeSettings(
             Map.of(
+                FeatureFlags.IDENTITY,
+                true,
+                SECURITY_SYSTEM_INDICES_PERMISSIONS_ENABLED_KEY,
```

Comment:
This is from a previous commit that can be removed now. In a previous iteration I created an in-memory role with a SystemIndexPermission, but have since updated the `SystemIndexAccessEvaluator` to check if the current user is a pluginUser. The SystemIndexAccessEvaluator logic will always kick in for a pluginUser regardless of what the action being performed is.